pom.xml

```diff
@@ -63,8 +63,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.1</version>
 				<configuration>
-					<source>1.7</source>
-					<target>1.7</target>
+					<source>8</source>
```

Comment:
1.8?

Reply:
I think this got updated automatically by something, I just reversed it.